src/SVector.jl

```diff
@@ -36,10 +36,6 @@ const SVector{S, T} = SArray{Tuple{S}, T, 1, S}
 ## SVector methods ##
 #####################
 
-@propagate_inbounds function getindex(v::SVector, i::Int)
-    v.data[i]
-end
```

Comment:
Haha, this redundant method has been in here for years. I removed it in favour of the linear getindex from `SArray`